frontend/apps/app/components/SessionDetailPage/components/Chat/Chat.tsx

```diff
@@ -48,7 +49,7 @@ export const Chat: FC<Props> = ({
     }
     onMessageSend(optimisticMessage)
 
-    await sendChatMessage({
+    await start({
```

Comment:
On a slightly different note, there should be an additional process to retrieve past messages, just as timeline_items was retrieved in the Server Component. However, I believe this can be done as follows.

https://langchain-ai.github.io/langgraphjs/concepts/persistence/#get-state